docs/schema/shapes/polystar.json

```diff
@@ -24,7 +24,6 @@
                     "$ref": "#/$defs/animated-properties/value"
                 },
                 "os": {
-                    "title": "Outer Roundness",
                     "title": "Outer Roundness as a percentage",
```

Comment:
I think here would be better to leave the first title and change the second into a `description`